lib/topological_inventory/openshift/operations/core/task_update.rb

```diff
@@ -0,0 +1,14 @@
+module TopologicalInventory
+  module Openshift
+    module Operations
+      module Core
+        module TaskUpdate
```

Comment:
@eclarizio The name seems to be repeating TaskUpdate and update_task. Should we call the module Task and the method update?

Reply:
I feel like that might be confusing since we have an actual `Task` model already, plus this is just a mixin that's housing the `update_task` method, so I think it's probably ok to leave it.

Reply:
@eclarizio I don't think there is enough going on here to warrant its own module